### DISCUSSION TITLE
How to encode a PNG (lossless) to JPEG?

### DISCUSSION THREAD

**Discussion by S3Y3D** [InterDigitalInc/CompressAI#94] [Q&A] [answered]
Hi guys,
I need to convert Kodak dataset PNG files into JPEG files at specific bitrate to see the results.
How can I do that using CompressAI?

Thanks 🌻

**Comment by harleyzhang** [comment, marked as answer]
You don't need compress AI. Any jpeg encoder can do that, for example, JPEG XT. https://jpeg.org/jpegxt/software.html

**Reply by S3Y3D** [reply]
Thanks @harleyzhang 🌻
But which one is the standard JPEG encoder? JPEG 1, JPEG XL, JPEG XS, JPEG XT , ... ?
I mean the one used in CompressAI's Image below.

![kodak-psnr (Phone)](https://user-images.githubusercontent.com/59525413/134842781-2a9cac88-61ac-4729-b9f6-9b3b644c4e0d.png)

**Reply by fracape** [reply]
That is basic JPEG that ships with Pillow. The codecs referenced in this figure are run using compressai.utils.bench. In particular JPEG [here](https://github.com/InterDigitalInc/CompressAI/blob/278f45e6f444448d3c44d973e51f9723768afa18/compressai/utils/bench/codecs.py#L209)

**Reply by S3Y3D** [reply]
Thanks 🌻
